pkg/controller/etcdproxy/etcdproxy_controller.go

```diff
@@ -414,3 +446,104 @@ func (c *EtcdProxyController) handleObject(obj interface{}) {
 		return
 	}
 }
+
+// createEtcdProxyServerCACerts creates ConfigMap in controller namespace with Etcd Proxy CA certificate.
+func (c EtcdProxyController) createEtcdProxyServerCACerts(etcdstorage *etcdstoragev1alpha1.EtcdStorage) error {
+	// ConfigMap in controller namespace for the etcd proxy CA certificate.
+	caConfigMap, err := c.kubeclientset.CoreV1().ConfigMaps(c.config.ControllerNamespace).
+		Get(etcdProxyCAConfigMapName(etcdstorage), metav1.GetOptions{})
+	if errors.IsNotFound(err) {
+		data := make(map[string]string, 1)
```

Comment:
TODO: Better map handling.

Reply:
In this case, I think this is a good way to do this, so leaving it like this for now.

Reply:
`data := map[string]string{"ca.pem": EtcdProxyCACert}`